internal/commands/config_pull_policy.go

```diff
@@ -0,0 +1,81 @@
+package commands
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	pubcfg "github.com/buildpacks/pack/config"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/logging"
+)
+
+func ConfigPullPolicy(logger logging.Logger, cfg config.Config, cfgPath string) *cobra.Command {
+	var unset bool
+
+	cmd := &cobra.Command{
+		Use:     "pull-policy <if-not-present | always | never>",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "List, set and unset the global pull policy used by other commands",
+		Aliases: []string{"pull-policy"},
```

Comment:
```suggestion
		Aliases: []string{"policy"},
```
We only need to set aliases for commands that aren't the typical command use; in this case, cobra selects the first word of `Use` as the command, so there is no need to set it again in `Aliases`. I would set `policy` there, though, to try and make it simpler for users